### PR TITLE
Various template improvements

### PR DESCRIPTION
**Pull request by gunnaraasen**
- Adds a contributing blurb on all docs pages for the current version of a product.
- Adds a warning message at the top of all old product versions.
- All links to product pages now default to the product index page for the current version.
- InfluxDB overview page has been moved to the v0.9 index page and removed from the menu.

### CODE REVIEW

File: content/influxdb/v0.9/index.md [hunk 1]
```diff
@@ -7,3 +7,23 @@ menu:
...
+Here are some of the features that InfluxDB currently supports that make it a great choice for working with time series data.
+
+* Written entirely in Go. It compiles into a single binary with no external dependencies.
+* Clustering is built in. Nothing else is needed to make data highly available (unlike Redis, ZooKeeper, Cassandra, and others).
+* Simple, high performing write and query HTTP(S) APIs.
+* Plugins support for other data ingestion protocols such as Graphite, collectd, and OpenTSDB.
+* Expressive SQL-like query language tailored to easily query aggregated data.
+* Tags allow series to be indexed for fast and efficient queries.
+* Retention policies efficiently auto-expire stale data.
+# Continuous queries automatically compute aggregate data to make frequent queries more efficient.
```
Comment (rkuchan) [inline content/influxdb/v0.9/index.md]: Just a formatting thing: `*` instead of `#`

**Comment by rkuchan** [comment]
Looks about ready to go, just a couple things:

* On the landing pages, the `open source` link in the contributing blurb goes to a 404 page.
* In the InfluxDB docs, when I click on `Introduction` and then the `Overview` link, `Overview` appears in the sidebar. When I click `Installation`, `Overview` disappears from the sidebar again.

**Comment by gunnaraasen** [comment]
Thanks @rkuchan for pointing those out. Just updated the PR to address them. Anything else look funky?

**Comment by rkuchan** [comment]
Looks good to me! 👍

**Pull request merged**